### PR TITLE
Add .to/.cuda/cpu to KNoise

### PR DESCRIPTION
**Pull request by fzimmermann89**

### CODE REVIEW

**Comment by fzimmermann89** [comment]
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #270
* #272
* #269
* #268
* #267
* #266
* #265
* #264
* __->__ #263

**Review by ckolbPTB — APPROVED** [review]

**Comment by ckolbPTB** [comment]
Can you please add a cuda test for KNoise?

File: tests/data/test_knoise.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    assert noise.data.is_cuda
+
+
+@pytest.mark.cuda()
+def test_KNoise_cpu(random_test_data):
+    """Move KNoise object to CUDA memory and back to CPU memory."""
+    noise_cuda = KNoise.from_tensor_and_kheader(data=random_test_data).cuda()
+    noise_cpu = noise_cuda.cpu()
+    assert noise_cpu.data.is_cpu
+    assert noise_cuda.data.is_cpu
```
Comment (ckolbPTB) [inline tests/data/test_knoise.py]: doesn't .cuda() and .cpu() create copies of the data? so shouldn't noise_cuda.data be still on cuda?

Comment (fzimmermann89) [reply tests/data/test_knoise.py]: this tests was completly wrong ,even more than you thought :D

**Pull request merged**